configs/katello/3.9.yaml

```diff
@@ -0,0 +1,270 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+  :3.8.0:
+    :redmine_version_id: 356
+  :3.7.0:
+    :redmine_version_id: 264
+  :3.7.1:
+    :redmine_version_id: 355
+:releases:
+  :3.9.0:
+    :redmine_version_id: 899
+:code_name: New England IPA
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-3.8
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :katello-installer:
+    :branch: KATELLO-3.8
+    :repo: https://github.com/Katello/katello-installer.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/1.19
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :puppet-katello:
+    :branch: 8.0.2
```

Comment:
We don't know yet which versions will be used for 3.9. I'd leave them out until we do.

Reply:
will do